Sources/PaymentsCore/Networking/Models/Requests/ConfirmPaymentSourceRequest.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+/// Describes request to confirm a payment source (approve an order)
+public struct ConfirmPaymentSourceRequest: APIRequest {
```

Comment:
qq: Should this be visible to the merchant?

Reply:
good call, I agree this shouldn't be public!!!
But we had to since this request lives in `PaymentsCore`, but `Card` module (and others in the future) also need it :(

Reply:
Ah yeah it'd be cool if there was a way to have it live in `Card` but sometimes the modules line up that way.

Reply:
Why does this need to live in Core? It should only be relevant to Card integrations.

Reply:
since we can do `confirm-payment-source` with payment sources other than card, I think this should live in `Core` so other payment methods we add in the future can use it :smile:

Reply:
🤔  At this point, it is only being used for card checkouts - so it should live in the Card Module. We want to avoid the PaymentsCore being a catch-all for everything and utilize / lean into the modularity we have. If we come across other payment methods needing `confirm-payment-source` we can cross that bridge later. How's that sound?

Reply:
yea that works for me!!

Reply:
actually thinking a bit about it, I think `confirm-payment-source` makes more sense living in Core. a few reasons:
- it's basically just an API wrapper around an Orders v2 endpoint (and we know all endpoints work for all payment methods), plus I believe handling network requests should be Core's responsibility.
- Why do we want to move it to Card when we know for sure it works for other payment methods and we'll have to move it back to Core in the future?

Please lemme know your thought :smile: